src/index.html

```diff
@@ -17,11 +17,376 @@
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="mac photo"
+          class="card__photo"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rate">
+          <div class="card__rate-stars card__rate-stars--4">
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+          </div>
+          <div class="card__rate-text">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          <span class="card__button-text">Buy</span>
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="mac photo"
+          class="card__photo"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rate">
+          <div class="card__rate-stars card__rate-stars--4">
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+          </div>
+          <div class="card__rate-text">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
```

Comment:
There is an inconsistency in the `data-qa` attribute naming. In previous cards, it is named `data-qa="card-hover"`, but here it is just `data-qa="hover"`. Ensure consistency in attribute naming for easier maintenance and readability.